megablocks/layers/dmoe_test.py

```diff
@@ -146,8 +146,11 @@ def testdMoE_ForwardVersusBaseline(self, bs, sl, hs):
     def testdMoE_ForwardVersusMoE(
             self, bs, sl, hs, num_experts, top_k,
             num_input_bits=-1, num_remat_bits=-1):
-        if num_input_bits > 0 or num_remat_bits > 0:
+        if num_input_bits > 0:
```

Comment:
Can we just adjust the tolerance on allclose instead of bailing on the test?

Reply:
So this is actually just skipping redundant test cases because whether we quantize the input or not doesn't affect any of the asserts here. Made the comment in the next line more explicit about this.

Reply:
Can we change the configs we pass to this test? This is kind of a nit, but I think that's the proper way to do it.

Reply:
Done. The `_FORWARD_TESTS` are now split into more granular subsets.